docs/api-reference/static-analysis.md

```diff
@@ -40,109 +45,219 @@ npm install --save-dev @dbos-inc/eslint-plugin
+      baseDirectory: __dirname,
+      recommendedConfig: js.configs.recommended
+    });
+
+    module.exports = [
+      ...compat.extends("plugin:@dbos-inc/dbosRecommendedConfig"),
+
+      { plugins: { "@dbos-inc": dbosIncEslintPlugin } },
+
+      {
+        languageOptions: {
+          parser: typescriptEslintParser,
+          parserOptions: { project: "./tsconfig.json" },
+          globals: { ...globals.node, ...globals.es6 }
+        }
+      },
+
+      { rules: {} }
+    ];
+    ```
+
+    If you'd like stronger typing, you can use `typescript-eslint`'s [`config`](https://typescript-eslint.io/packages/typescript-eslint/#config) helper function.
+    First, run `npm install --save-dev typescript-eslint`, and then set up your config like this:
+
+    ```js
+    const { FlatCompat } = require("@eslint/eslintrc");
+    const dbosIncEslintPlugin = require("@dbos-inc/eslint-plugin");
+    const typescriptEslint = require("typescript-eslint");
+    const typescriptEslintParser = require("@typescript-eslint/parser");
+    const globals = require("globals");
+    const js = require("@eslint/js");
+
+    const compat = new FlatCompat({
+        baseDirectory: __dirname,
+        recommendedConfig: js.configs.recommended
+    });
+
+    module.exports = typescriptEslint.config(
+      {
+        extends: compat.extends("plugin:@dbos-inc/dbosRecommendedConfig"),
+        plugins: { "@dbos-inc": dbosIncEslintPlugin },
+
+        languageOptions: {
+            parser: typescriptEslintParser,
+            parserOptions: { project: "./tsconfig.json" },
+            globals: { ...globals.node, ...globals.es6 }
+        },
+
+        rules: { }
+      }
+    );
+    ```
+  </TabItem>
+  <TabItem value="legacy-config" label="Legacy config">
+    <h4> This config style will work with ESLint 8 and below. </h4>
+    Place an `.eslintrc` file similar to the following in your project directory:
+
+    ```json
+    {
+      "root": true,
+      "extends": [
+        "plugin:@dbos-inc/dbosRecommendedConfig"
+      ],
+      "plugins": [
+        "@dbos-inc"
+      ],
+      "env": {
+        "node": true,
+        "es6": true
+      },
+      "rules": {
+      },
+      "parser": "@typescript-eslint/parser",
+      "parserOptions": {
+        "project": "./tsconfig.json"
+      }
+    }
+    ```
+  </TabItem>
+</Tabs>
 
 The example above configures the project for the recommended `eslint` configuration.  Adjust the `extends`, `rules`, `plugins`, `env`, and other sections as desired, consulting the configurations and rules below.
 
 Finally, to make `eslint` easy to run, it is suggested to place commands in `package.json`.  For example:
-```json
-  "scripts": {
-    "build": "tsc",
-    "test": "...",
-    "lint": "eslint src",
-    "lint-fix": "eslint --fix src"
-  },
 
+```json
+"scripts": {
+  "build": "tsc",
+  "test": "...",
+  "lint": "eslint src",
+  "lint-fix": "eslint --fix src"
+}
 ```
 
 #### Profiles to extend
```

Comment:
Drop this section now that we only have one profile

Reply:
Resolved